packages/design-system-web/src/components/geov-entity-props-by-predicate/geov-entity-props-by-predicate.tsx

```diff
@@ -179,32 +176,27 @@ export class GeovEntityPropsByPredicate {
   }
 
   componentDidLoad() {
-    const shadowRoot = this.hostElement.shadowRoot;
-    this.resizePage(shadowRoot);
+    this.resizePage();
     window.addEventListener('resize', this.handleResize);
   }
 
-  resizePage(shadowRoot: ShadowRoot) {
+  resizePage() {
     // Display Expand button if need. The page must be loaded in order to have the measurements
-    if (shadowRoot) {
-      const literalContainer = shadowRoot.querySelector('.literal-container');
-      const itemButton = shadowRoot.querySelector('.literal-container ion-button') as HTMLElement;
-
-      if (literalContainer && itemButton) {
-        const labelLiteral = literalContainer.querySelector('ion-label');
-        if (labelLiteral) {
-          // If size of text > size of container
-          if (labelLiteral.scrollWidth > labelLiteral.clientWidth) {
-            itemButton.style.display = 'block';
-          }
+    if (this.literalContainer && this.itemButton) {
+      const labelLiteral = this.literalContainer.querySelector('ion-label');
```

Comment:
Is maintained because otherwise, by reference, we have 0 as the value for clientWidth and scrollWidth...

Reply:
Done with PR 91, thank you